src/main/java/org/gridsuite/gateway/filters/SupervisionAccessControlFilter.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.gateway.filters;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.cloud.gateway.filter.GatewayFilterChain;
+import org.springframework.core.Ordered;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.server.ServerWebExchange;
+import reactor.core.publisher.Mono;
+
+/**
+ * @author Achour BERRAHMA <achour.berrahma at rte-france.com>
+ */
+@Component
+public class SupervisionAccessControlFilter extends AbstractGlobalPreFilter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SupervisionAccessControlFilter.class);
+    private static final String SUPERVISION_PATH = "supervision";
+    public static final String ACCESS_TO_SUPERVISION_ENDPOINT_IS_NOT_ALLOWED = "{}: 403 Forbidden, Access to supervision endpoint is not allowed";
+
+    @Override
+    public Mono<Void> filter(ServerWebExchange exchange, GatewayFilterChain chain) {
+        LOGGER.debug("Filter: {}", getClass().getSimpleName());
+
+        String path = exchange.getRequest().getURI().getPath();
+        if (path.toLowerCase().contains(SUPERVISION_PATH)) {
```

Comment:
That seem to simple for me because it will match all paths that contains "supervision", not the path "/*/v*/supervision".  
For example "/server/v1/studies/supervision_network_20240101" would be blocked.

Reply:
you can do a match with `/[^/]+/v\d+/supervision(?=/|$)`